cli/src/pcluster/config/config_patch.py

```diff
@@ -171,45 +220,6 @@ def update_policy_level(self):
             else UpdatePolicy.SUPPORTED.level
         )
 
-    def _create_default_section(self, config, section):
```

Comment:
How missing parameters/sections are now compared against defaults?

Reply:
There are no more defaults to skip. We're comparing the two config files rather than the two config objects.